### PR TITLE
Some late feedback on 'Contribute to Guide' PR #109

### PR DESCRIPTION
**Pull request by danielnordh**
I was too slow to comment on it before the merge but here are some tweaks.
Apart from fixed typos and some simplified language the main changes are:

- Added Slack channel and Github sections on contribute.md
- Suggest changing the nav title to 'Contribute to guide' to differentiate from higher level 'Contribute'
- Bump sub-headline styling to differentiate do's/don't

### CODE REVIEW

**Comment by danielnordh** [comment]
If/When this PR gets merged, please squash the commits as there it is a bit 'unclean', containing commits already merged.
Discussed this with @pavlenex and it seems to be the best way.

Reason: I branched off a PR that was already merged (unbeknownst to me). I then merged master into that branch to make sure there were no conflicts. But the previous commits still show up on this PR.

**Review by pavlenex — APPROVED** [review]
Just a note, since this PR is based of my earlier PR and not the master, it would  be a good idea to squash commits when merging this PR.

**Review by pavlenex — APPROVED** [review]
Thanks for quickly addressing the navigation and for all the improvements to the page.

File: guide/contribute/contribute.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 layout: guide
-title: Contribute
+title: Contribute to Guide
```
Comment (GBKS) [inline guide/contribute/contribute.md:3]:
```suggestion
title: Contribute to guide
```

File: guide/contribute/contribute.md [hunk 2]
```diff
@@ -16,6 +16,18 @@ This page shows you how to get involved and start contributing.
...
+
+We use the #bitcoin-design-guide channel in the [Bitoin Design Slack community]({{ site.slack_invite_url }}) to communicate and discuss work on guide. To hear what is going on, see what you can help with, or suggest new contributions please come and say hi.
+
+---
+
+**[Github](https://github.com/BitcoinDesign/Guide)**
+
+We use Github to host the content of the guide. [Issues](https://github.com/BitcoinDesign/Guide/issues) are used to post ideas for improvements, and proposed changes are posted as [Pull Requests](https://github.com/BitcoinDesign/Guide/pulls). 
+For some ideas we use [Discussions](https://github.com/BitcoinDesign/Guide/discussions). 
+---
```
Comment (GBKS) [inline guide/contribute/contribute.md:29]:
```suggestion

---
```

File: guide/contribute/contribute.md [hunk 3]
```diff
@@ -16,6 +16,18 @@ This page shows you how to get involved and start contributing.
 
 ---
 
+**[Slack channel]({{ site.slack_invite_url }})**
+
+We use the #bitcoin-design-guide channel in the [Bitoin Design Slack community]({{ site.slack_invite_url }}) to communicate and discuss work on guide. To hear what is going on, see what you can help with, or suggest new contributions please come and say hi.
```
Comment (GBKS) [inline guide/contribute/contribute.md:21]:
```suggestion
We use the #bitcoin-design-guide channel in the [Bitcoin Design Slack community]({{ site.slack_invite_url }}) to communicate and discuss work on guide. To hear what is going on, see what you can help with, or suggest new contributions please come and say hi.
```

File: guide/contribute/contribute.md (hunk 1 above)
Comment (danielnordh) [reply guide/contribute/contribute.md:3]: This makes sense.
Does this mean we need to change: parent: .... as well for navigation to work?

**Review by GBKS — APPROVED** [review]

**Review by johnsBeharry — APPROVED** [review]

**Pull request merged**